src/test/java/edu/ucsb/mapache/controllers/SlackSlashCommandControllerTests.java

```diff
@@ -0,0 +1,183 @@
+package edu.ucsb.mapache.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import edu.ucsb.mapache.config.SecurityConfig;
+import edu.ucsb.mapache.repositories.ChannelRepository;
+
+import org.springframework.http.MediaType;
+
+
+@WebMvcTest(value = SlackSlashCommandController.class)
+@Import(SecurityConfig.class)
+public class SlackSlashCommandControllerTests {
+  
+  @Autowired
+  private MockMvc mockMvc;
+  
+  @Autowired
+  SlackSlashCommandController slackSlashCommandController;
+
+  @MockBean
+  ChannelRepository channelRepository;
+
+  private final String URL="/api/public/slash-command";
```

Comment:
Super nitpicky, but it might be a good idea to avoid using `URL` as a constant name since it's also the name of a class in the Java standard library. Maybe a more specific name would be more readable as well?

Reply:
yep, just changed URL to testURL.

Reply:
But I still see URL in the code?

Reply:
Ok.. I see it was changed on a different branch, in a different pull request.

This PR should probably be closed and we can just merge the other one.